gapic/src/main/com/google/gapid/perfetto/models/FrameEventsTrack.java

```diff
@@ -291,6 +343,9 @@ public void getRange(Consumer<TimeSpan> span) {
       }
     }
 
+    public FrameSelection getSelection() {
+      return new FrameSelection(frameNumbers, layerNames);
```

Comment:
This function will be queried every time rendering the track, so it would be good to keep it efficient. It seems that frameNumbers and layerNames are fixed at slice creation time, maybe we can save a fixed FrameSelection at slice creation time as well? So as to avoid creating a new object every time. Or maybe I'm missing something here.

Reply:
This function will be called only when the event is selected. Storing the FrameSelection within the Slice would be a waste because the slices are created anew for every redraw.

Reply:
Hmmm from my understanding, if following the chain of FrameEventSummaryPanel.java -> renderTrack -> renderSlices -> getSelectedFrames -> getSelection -> new FrameSelection, it's called at every track rendering time?

Reply:
Got it. Done.